src/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ if (WIN32)
                        ${DLL_LOCATION} ${UNIT_TEST_DIR})
 else(WIN32)
     find_package(LibXml2 2.9 REQUIRED)
-    find_package(CURL 7.31 REQUIRED)
+    find_package(CURL 7.29 REQUIRED)
```

Comment:
7.29 is from 2/2013.  7.31 is only 4 months newer.  The current release is 7.52.  The current debian distributed version is 7.47.  If anything I'd prefer to move to a newer version.